src/components/shared/Entity/prompts/steps/dataFlowReset/Publish/index.tsx

```diff
@@ -19,6 +18,7 @@ function PublishStepDataFlowReset() {
 
     const stepIndex = useLoopIndex();
     const stepIsIdle = useStepIsIdle();
+    const incompatibleCollectionChecker = useIncompatibleCollectionChecker();
```

Comment:
My preference is to always have functions start with an active verb. There are some exceptions that are in the codebase today, like the instance of `publicationHandler` in this file (which should, at the very least, be `handlePublication` IMO). That said, `checkIncompatibleCollections`, `evaluateIncompatibleCollections`, or something along those lines would be preferred.

Reply:
I can rename - but I don't think functions should always start with verbs. Especially with stores/hooks and being able to alter when something is a function vs being a computed value. Keeping the verb at the end does help keep all the hooks more closely named based on the `incompatibleCollection` portion of the name.